inc/event_queue.h

```diff
@@ -1,10 +1,12 @@
 #pragma once
 #include <stdint.h>
 #include <stdbool.h>
+#include "objpool.h"
```

Comment:
I'd like to propose system headers and regular headers should have a newline between them, same with the pragma guards and system header. This follows the standard Google C++ include rules also headers should be in alphabetical order just makes it easier for readability when lots of includes are present. If you are okay with this I will add it to the style guide and linter.